tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -534,6 +534,124 @@ def test_calculated_summary_value_source(mocker, list_item_id):
     )
 
 
+@pytest.mark.parametrize(
+    "list_item_id",
+    [None, "item-1"],
+)
+def test_new_calculated_summary_value_source(mocker, list_item_id):
+    schema = mocker.MagicMock()
+    schema.get_block = Mock(
+        return_value={
+            "id": "number-total",
+            "type": "CalculatedSummary",
+            "calculation": {
+                "operation": {
+                    "+": [
+                        {"source": "answers", "identifier": "number-answer-1"},
+                        {"source": "answers", "identifier": "number-answer-2"},
+                    ]
+                }
+            },
+        },
+    )
+
+    location = (
+        Location(
+            section_id="test-section", block_id="test-block", list_item_id="item-1"
+        )
+        if list_item_id
+        else Location(
+            section_id="test-section", block_id="test-block", list_item_id=None
+        )
+    )
```

Comment:
You have the list item already so don't need the `if`?
```suggestion
    location = Location(
    	section_id="test-section", block_id="test-block", list_item_id=list_item_id
    )
```

Reply:
Fixed 👍